Arask/sheet.json

```diff
@@ -0,0 +1,8 @@
+{
+	"html": "arask.html",
+	"css": "arask.css",
+	"authors": "Taliyah Larson, Westley Hindley",
+	"roll20userid": "1605986",
+	"preview": "preview.jpg",
```

Comment:
The preview needs to exactly match the image included. The .jpg needs to be updated to .JPG.

Reply:
Change has been implemented!